onnx/test/cpp/function_context_test.cc

```diff
@@ -149,7 +149,265 @@ void RegisterCustomFunctionSchema() {
   (void)unused;
 }
 
-// Test for Context dependant function with type context
+void RegisterVersionedLogSoftmaxFunctionSchema(const std::string& op_type, bool with_missing_version) {
```

Comment:
Is this just testing the ability to register two function builders? May be adding a comment to indicate what is being tested will be helpful.

Reply:
Ok, never mind. I guess it is just a utility.